### PR TITLE
Fix

### PR DESCRIPTION
**Pull request by maropu**

### CODE REVIEW

**Comment by dongjoon-hyun** [comment]
Great!

File: sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/dsl/package.scala [hunk 1]
```diff
@@ -110,12 +110,12 @@ package object dsl {
...
     def isNotNull: Predicate = IsNotNull(expr)
 
-    def isTrue: Predicate = BooleanTest(expr, Some(true))
-    def isNotTrue: Predicate = Not(BooleanTest(expr, Some(true)))
-    def isFalse: Predicate = BooleanTest(expr, Some(false))
-    def isNotFalse: Predicate = Not(BooleanTest(expr, Some(false)))
-    def isUnknown: Predicate = BooleanTest(expr, None)
-    def isNotUnknown: Predicate = Not(BooleanTest(expr, None))
+    def isTrue: Predicate = IsTrue(expr)
+    def isNotTrue: Predicate = IsFalse(expr)
```
Comment (beliefer) [inline sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/dsl/package.scala]: `isNotTrue` != `IsFalse`

File: sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/dsl/package.scala [hunk 2]
```diff
@@ -110,12 +110,12 @@ package object dsl {
...
+    def isFalse: Predicate = IsFalse(expr)
+    def isNotFalse: Predicate = IsTrue(expr)
```
Comment (beliefer) [inline sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/dsl/package.scala]: `isNotFalse` != `IsTrue`

File: sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/dsl/package.scala (hunk 1 above)
Comment (maropu) [reply sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/dsl/package.scala]: yea, we need more fixes.

**Pull request closed without merging**